packages/contracts/test/utils/index.ts

```diff
@@ -0,0 +1,205 @@
+import { AppIdentity } from "@counterfactual/types";
+import * as chai from "chai";
+import { solidity } from "ethereum-waffle";
+import { defaultAbiCoder, keccak256, solidityPack } from "ethers/utils";
+
+export const expect = chai.use(solidity).expect;
+
+export enum AssetType {
+  ETH,
+  ERC20,
+  ANY
+}
+
+export class App {
```

Comment:
```suggestion
export class AppInstance {
```

Reply:
This class actually should probably re-use the model in the machine package. I wrote it before I wrote the other one.